cloud/src/meta-service/meta_service.cpp

```diff
@@ -1682,6 +1682,29 @@ static bool check_delete_bitmap_lock(MetaServiceCode& code, std::string& msg, st
     return true;
 }
 
+static bool check_and_remove_delete_bitmap_update_lock(
```

Comment:
why add this method, not do it in `remove_delete_bitmap_update_lock` directly?

Reply:
already change it